x-pack/plugins/monitoring/public/components/cluster/overview/index.js

```diff
@@ -13,23 +13,32 @@ import { BeatsPanel } from './beats_panel';
 
 import { EuiPage, EuiPageBody } from '@elastic/eui';
 import { ApmPanel } from './apm_panel';
+import { UNLINKED_DEPLOYMENT_CLUSTER_UUID } from '../../../../common/constants';
 
 export function Overview(props) {
+  const isFromUnlinkedDeployment = props.cluster.cluster_uuid === UNLINKED_DEPLOYMENT_CLUSTER_UUID;
```

Comment:
This logic appears to be appropriately reused in a bunch of places. Is the thought here that used a predefined constant is better than `undefined` because it ends up in the global scope, and we don't want to add logic to the global scope handling?

(I'm asking to better understand your reasoning rather than "this is not right!")

Reply:
Yup basically. I didn't want to use an empty string for `cluster_uuid` and I felt something like `undefined` or `null` misrepresented what's going on.

Reply:
> I felt something like `undefined` or `null` misrepresented what's going on.

But looking at the name, `cluster_uuid`, I feel like that's exactly what it means though?

I do think one problem with using `undefined`/`null` in the short term is that we redirect to the cluster listing page if multiple clusters are detected and it's not supplied / found. I wouldn't want you to change that behavior in this PR, but I do think we should stop redirecting automatically from the cluster listing page, which would then open up using it for that purpose.